Services/DataX.Flow/DataX.Flow.InteractiveQuery/InteractiveQueryManager.cs

```diff
@@ -29,6 +30,7 @@ public class InteractiveQueryManager
         private const string _DataBricks = Config.ConfigDataModel.Constants.SparkTypeDataBricks;
         private readonly string _sparkType = _HDInsight;
         private readonly IConfiguration _configuration;
+        private static readonly Regex _OpsPath = new Regex(@"wasbs:\/\/(.*)@(.*).blob.core.windows.net\/(.*)$", RegexOptions.IgnoreCase);
```

Comment:
can this be just a local property in this method ConvertToDbfsFilePath?

Reply:
done